vanilla-framework/vanilla-framework
# Update table component to match spec

## Done

Update table component to match spec

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/base/table/
- Verify it matches [the spec](https://github.com/ubuntudesign/vanilla-design/tree/master/Table)

## Details

Fixes #1306

## Screenshots

@kwm14 The trouble with right aligning numerical cells is the disparity with textual header labels on large screens - thoughts?

<img width="1440" alt="screen shot 2017-09-28 at 11 35 10" src="https://user-images.githubusercontent.com/505570/30962489-d4cb6a6c-a441-11e7-891c-8010d5431d1c.png">
